libraries/botbuilder-dialogs/etc/botbuilder-dialogs.api.md

```diff
@@ -254,6 +253,7 @@ export abstract class DialogContainer<O extends object = {}> extends Dialog<O> {
     readonly dialogs: DialogSet;
     findDialog(dialogId: string): Dialog | undefined;
     protected getInternalVersion(): string;
+    telemetryClient: BotTelemetryClient;
```

Comment:
I am unsure if this move is needed, or just accidentally got in.

Reply:
This is a side-effect of regenerating the API report for the library; it wasn't an intentional change, but this line would eventually be added when the report is regenerated.